financial-connections/src/main/java/com/stripe/android/financialconnections/domain/CreateInstantDebitsResult.kt

```diff
@@ -26,13 +26,22 @@ internal class RealCreateInstantDebitsResult @Inject constructor(
         bankAccountId: String,
     ): InstantDebitsResult {
         val consumerSession = consumerSessionProvider.provideConsumerSession()
+
         val clientSecret = requireNotNull(consumerSession?.clientSecret) {
             "Consumer session client secret cannot be null"
         }
 
+        val billingEmailAddress = requireNotNull(consumerSession?.emailAddress) {
+            "Consumer session email address cannot be null"
+        }
```

Comment:
super nit - maybe `billingEmail` to avoid confusion with `billingAddress`?

Reply:
I’m renaming `billingAddress` to `billingDetails`, so should remove any opportunity for confusion.